src/msg/async/ProtocolV2.cc

```diff
@@ -1117,8 +1132,8 @@ void ProtocolV2::decrypt_payload(char *payload, uint32_t &length) {
 void ProtocolV2::calculate_payload_size(uint32_t length, uint32_t *total_len,
                                         uint32_t *sig_pad_len,
                                         uint32_t *enc_pad_len) {
-  bool is_signed = sign_frames();
-  bool is_encrypted = encrypt_frames();
+  bool is_signed = auth_meta->is_mode_secure(); // REMOVE ME
```

Comment:
Uh...?
If you tag it FIXME it'll at least show up more easily in the future...

Reply:
this code is changing already